### PR TITLE
Add functionality to prepopulate the form from JSON data

### PR DESCRIPTION
**Pull request by darkwing**
Added stub data, button to prepopulate the form based on multiple plans and steps.

Marking as *not ready* because I want to pretty up the code.  It's...got room for improvement.

### CODE REVIEW

**Comment by darkwing** [comment]
Wanna have a look at this @pdehaan ?  I'd love ot know what you think.

File: client/static/assets/scripts/configs.js [hunk 1]
```diff
@@ -9,7 +9,9 @@ angular.module('LoadsApp')
       .when('/cluster', route('ClusterManagementController', 'cluster'))
       .when('/agents/status', route('AgentsStatusController', 'agents-status'))
       .when('/agents/check', route('LaunchAgentHealthCheckController', 'agents-check'))
-      .when('/tests', route('TestsController', 'tests'))
+      .when('/projects', route('ProjectsController', 'projects'))
+      .when('/projects-builder', route('ProjectBuilderController', 'projects-builder'))
+      .when('/projects-builder/:id?', route('ProjectBuilderController', 'projects-builder'))
```
Comment (darkwing) [inline client/static/assets/scripts/configs.js]: It seems like I'm getting a 404 when I go to "/projects-builder/33" -- any reason why it wouldn't just load the form?  I find this really odd.

**Comment by pdehaan** [comment]
@darkwing Looks good... How do I test this?
I can generate the JSON and have it display in the `<textarea>`, but it doesn't seem to save my projects under the `/#/projects` listing page.

File: client/static/assets/scripts/controllers.js [hunk 2]
```diff
@@ -13,8 +13,165 @@ angular.module('LoadsApp')
...
+          var stepItem = convertElementsToObject(jQuery(this).find(':input'));
+          planItem.steps.push(stepItem);
+        });
+
+        data.plans.push(planItem);
+      });
+
+      // Finally, focus on the generated code
+      projectJSON = data;
+      var textarea = jQuery('#project-tool-textarea').val(JSON.stringify(data)).get(0);
```
Comment (pdehaan) [inline client/static/assets/scripts/controllers.js]: If you want this to be "more pretty", you could tweak it to:
```js
JSON.stringify(data, null, 2)
```

Where the `2` is how many spaces to indent the elements. I think I filed this as #47 but then realized that we haven't merged this yet.

**Comment by darkwing** [comment]
Ugh, forgot I still had this out there.  Rebasing now.

**Pull request merged**